robocop/checkers/tags.py

```diff
@@ -170,6 +171,7 @@ class TagNameChecker(VisitorChecker):
     is_keyword = False
     reserved_tags = {
         "robot:exit",
+        "robot:flatten",
```

Comment:
At this point I wonder if Robot Framework store reserved tags in some constant we can use? Instead of updating our rule whenever there is new reserved tag.

Reply:
Good point, I will check.

Reply:
Haven't found it in the docs. In the API there's only this: https://robot-framework.readthedocs.io/en/stable/_modules/robot/model/tags.html

Reply:
Right, I see they're checking for tags depending on the current context. Ie https://github.com/robotframework/robotframework/blob/6a266bf852ad0525c350ea7412f7b33f81455e96/src/robot/running/context.py#L181. We need to stick to our current implementation.